pkgs/applications/networking/instant-messengers/teamspeak/client.nix

```diff
@@ -38,7 +38,7 @@ stdenv.mkDerivation rec {
       "http://teamspeak.gameserver.gamed.de/ts3/releases/${version}/TeamSpeak3-Client-linux_${arch}-${version}.run"
       "http://files.teamspeak-services.com/releases/${version}/TeamSpeak3-Client-linux_${arch}-${version}.run"
     ];
-    sha256 = if stdenv.is64bit 
+    sha256 = if stdenv.is64bit
```

Comment:
The fetchurl creates a seperate derivation for the network request, which is cached based on the hash and NOT the url. So always make sure to change the hash to some random hash when changing urls. Same for other fetch*